### DISCUSSION TITLE
Synapps Mobile

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#668] [Q&A]
Bonjour, 

Est-ce qu'il est prévu d'avoir un application qui afficherait les synapps sur tablette ou téléphone de manière full screen 
Avec des favoris etc car les navigateurs font perdre pas mal de place d'affichage

![image](https://github.com/witsa/synapps/assets/46195964/99146983-b2dd-4be3-a1f1-6a37efef0eac)

**Comment by WITNicolas** [comment]
Salut
Tu peux utiliser ce chemin :
http://IPREDY/web/ria/synapps/latest/index.html?lang=fr#/LABELSYNAP

Et a partir de là tu peux mettre ton favoris sur le smartphone ou tablette comme une WEBAPP. Et ça sera en fullscreen
En tout cas sous iphone c'est comme ça que ça fonctionne, sur android je sais pas...
